packages/js/uri-resolver-extensions/src/ExtendableUriResolver.ts

```diff
@@ -25,7 +25,7 @@ export class ExtendableUriResolver extends UriResolverAggregatorBase<
   // $start: ExtendableUriResolver-extInterfaceUri-static
   /** The default interface uri to which implementations should be registered */
   public static extInterfaceUri: Uri = new Uri(
-    "wrap://ens/uri-resolver.core.polywrap.eth"
+    "wrap://ens/wraps.eth:uri-resolver-ext@1.0.0"
```

Comment:
Good catch, we just chatted via DM and agree we should just rename version `1.1.0` to `1.0.0` since the changes are benign, and makes sense to have within a v1 release.